pyproject.toml

```diff
@@ -20,7 +20,7 @@ testino = "0.3.9"
 optional = true
 
 [tool.poetry.group.deployment.dependencies]
-django-gov-notify = "0.4.0"
+django-gov-notify = "0.5.0"
```

Comment:
There's a v0.6.0 as well - can we go to that?

Reply:
sure